pages/co-workingSpace/details.tsx

```diff
@@ -0,0 +1,168 @@
+import type { NextPage } from "next";
+import React from "react";
+import Head from "next/head";
+import GeneralLayout from "@/layouts/generalLayout";
+import Image from "next/image";
+import Office2 from "assets/co-workingSpace/Office2.png";
+import Office3 from "assets/co-workingSpace/Office3.png";
+import Office4 from "assets/co-workingSpace/Office4.png";
+import calender2 from "assets/icons/calender2.svg";
+import location from "assets/icons/location.svg";
+import verified from "assets/icons/verified.svg";
+import Button from "@/components/atoms/Button/Button";
+import Tabs from "@/components/Co-workingSpace/tabComp";
+import LocateMap from "@/components/Co-workingSpace/map"
+import { availableSpaceTabsData, availableSpacesTableData, businessHoursData, equipmentData, cateringData, facilitiesData, relaxZonesData, classicBasicsData } from "@/utils";
+import WorkSpaceTable from "@/components/Co-workingSpace/tableComp";
+import CheckBoxComp from "@/components/Co-workingSpace/checkBoxComp";
+
+const CoWorkingSpaceDetails: NextPage = () => {
+  const handleCheckboxChange = (selectedValues: string[]) => {
+    console.log('Selected values:', selectedValues);
+  };
+
+
+  return (
+    <div>
+      <Head>
+        <title>Afrisplash</title>
+        <meta name="description" content="Generated by create next app" />
+      </Head>
+      <GeneralLayout>
+        <div className="afri-container grid gap-8 xl:grid-cols-6 md:grid-cols-5 my-8">
+          <div className="relative md:h-[100%] h-[500px] rounded-lg bg-cover md:col-span-3 xl:col-span-4 bg-no-repeat bg-[url(../assets/co-workingSpace/Office1.png)]">
+            <div className="absolute md:pl-10 pl-4  flex flex-col justify-end rounded-lg inset-0 bg-gradient-to-t from-[#D6ECDC] via-transparent to-transparent">
+              <div className="flex justify-start mb-5 md:ml-14 ml-6">
+                <Image src={calender2} alt="chatIcon" className="mr-2" />
+                <h5 className="text-white text-center text-dark_black">Joined April 2022</h5>
```

Comment:
`text-dark_black` applies the same CSS properties as `text-white`.

please stick to one